### DISCUSSION TITLE
Reinsert the straight, simple connector which was recently removed

### DISCUSSION THREAD

**Discussion by PinoPair** [jgraph/drawio#2304] [Ideas]
The simple, straight connection without any label nor angles and curve was recently removed from the product. This is very annoying as either one has to manually remove the label or to straighten a curved line.

I just want to be able to draw a very simple straight line without labels nor curves nor angles like I could up to a couple of days ago. 

Also, we found a way to use them which is VERY annoying: one has to type "connector" in the search box, then click on "more results" and finally it appears. The ridiculous thing is that if you type "connection" it doesn't even appear. 

Please give us back the simple straight arrow. Thank you!

**Comment by davidjgraph** [comment]
It wasn't removed, show me a screenshot of your connector routing options menu.

**Reply by PinoPair** [reply]
It was removed from the "basic" shapes, now only connectors with labels or angles/curves appear. Take a look at the screenshot. The closest thing I can see in the application is the "connection" in the UML library which, by the way, has a different "pointer" shape, than the original "simple" straight arrow.

![image](https://user-images.githubusercontent.com/89016471/136545834-6daf33d2-a3b5-4b64-b22e-3e457c43caef.png)
![image](https://user-images.githubusercontent.com/89016471/136545881-55c3b1f2-c24f-4c97-8eed-29b41076a19c.png)


EDIT: Maybe I used the wrong wording, I am referring to this shape (take a look at picture below):

![image](https://user-images.githubusercontent.com/89016471/136546094-ef7e0815-0010-43dd-bb6e-a82981401df2.png)

**Comment by davidjgraph** [comment]
Connectors options are in the style panel when the connector is selected. Straight is the first option.

<img width="250" alt="Screenshot 2021-10-09 at 21 53 45" src="https://user-images.githubusercontent.com/1761087/136673505-68f2d1c3-b2a0-4409-8420-7cc00b653d68.png">

**Reply by PinoPair** [reply]
That's not what I am asking for. Up to last week I had the possibility to insert a simple, straight arrow directly from the library. Modifying the style of every arrow is very annoying, especially if you use many many arrows in every single diagram. I don't know, this looks like a basic function to me, the solution I found is to create one and then copy-paste it around which is not great. Honestly to me this is a big regression in terms of usability.

**Comment by davidjgraph** [comment]
What version are you using?

**Reply by PinoPair** [reply]
The online one, which is currently the 15.5.0

![image](https://user-images.githubusercontent.com/89016471/136700504-bb5f6239-5daa-40bd-8d5e-e3b5749033db.png)

**Comment by davidjgraph** [comment]
Issue added as https://github.com/jgraph/drawio/issues/2308

**Reply by PinoPair** [reply]
Thank you!

**Comment by PinoPair** [comment]
Thank you for resolving this issue, super fast!
